File: testdata/jennies/rawtypes/enums/JavaRawTypes/src/main/java/com/grafana/foundation/enums/Operator.java
package com.grafana.foundation.enums;


// This is a very interesting string enum.
public enum Operator {
    GREATER_THAN(">"),
    LESS_THAN("<");

    private String value;

    private Operator(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: testdata/jennies/rawtypes/enums/CSharpRawTypes/src/Grafana/Foundation/Enums/Operator.cs
namespace Grafana.Foundation.Enums;

using System.Runtime.Serialization;

// This is a very interesting string enum.
public enum Operator
{
    [EnumMember(Value = ">")]
    GreaterThan,
    [EnumMember(Value = "<")]
    LessThan
}
